apps/site-nft/src/components/Offsets/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { AppContainer } from 'components/AppContainers';
+import * as React from 'react';
+import { defineMessages } from 'react-intl';
+import { List, Image, Popup } from 'semantic-ui-react';
+import { PageHeader } from '../PageHeader';
+import icon from './images/icon_topup_big.svg';
+import randomhex from 'randomhex';
+import styles from './styles.module.less'
+
+const messages = defineMessages({
+  title: { defaultMessage: "NFT Offsets", description: "NFT Offsets listings page title" },
+  description: { defaultMessage: "Review the offsets each NFT has completed", description: "NFT Offsets listing description" },
+})
+
+const avatars = [
+  'https://react.semantic-ui.com/images/avatar/small/rachel.png',
```

Comment:
in the shared components we have our icons/avatars

Reply:
that would have been better - I didn't even think to look.

This is just throw-away code - so I will leave it like this if that's OK?